tensorflow/core/kernels/sparse/mat_mul_op.cc

```diff
@@ -951,7 +951,7 @@ class CSRSparseMatrixMatVec<GPUDevice, T> {
       const int n = a.dense_shape_host(1);
       const int nnz = a.values.size();
       DCHECK_EQ(nnz, a.col_ind.size());
-#if GOOGLE_CUDA && (CUDA_VERSION >= 10020)
+#if GOOGLE_CUDA && (CUDA_VERSION >= 10020) && !defined(_WIN32)
```

Comment:
These changes seem to undo #38802 for Windows. Is that intentional?

Reply:
Yes, it is intentional. SpMM is not defined on Windows for CUDA 10.2.